### DISCUSSION TITLE
OBS can't capture the game sound support by PlayCover

### DISCUSSION THREAD

**Discussion by gegejiawei** [PlayCover/PlayCover#283] [Q&A] [answered]
According to the online tutorial, I installed a virtual audio device(BlackHole 2ch), and a multi output device is created. However, the game sound cannot be captured by obs, while the voice of safari browser can be input into obs. Do other players have this problem? :)

**Comment by JoseMoreville** [comment, marked as answer]
Try using another tool, Blackhole is not supported, This should also be a question for blackhole than us.

**Reply by gegejiawei** [reply]
Loopback works, but it's not free.
